prow/crier/reporters/gerrit/reporter_test.go

```diff
@@ -28,13 +28,15 @@ import (
 
```

Comment:
Please add a test to verify this is working and prevent regressions. I think it is sufficient to concurrently call `Report` for a few different PJs that target the same gerrit change and use a fake gerrit client that modifies some change-specific state without any additional synchronization. Go's race detector should then catch any races.

Reply:
I have added a test in gerrit/reporter_test.sh for this purpose, where 20 threads competing for the same lock, would that do?